### PR TITLE
Split shift+click f range annotations changes

### PR DESCRIPTION
**Pull request by kelseiv**
Publishing Shift+click changes from [update for range annotation PR](https://github.com/influxdata/docs-v2/pull/2734)

### CODE REVIEW

File: content/influxdb/cloud/visualize-data/annotations.md [hunk 1]
```diff
@@ -21,21 +21,16 @@ Add annotations to your dashboards to provide useful, contextual information abo
...
 
-2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/).
-3. Click **Annotations**. The **Show Annotations** option is selected by default.
-4. Select the **Enable 1-Click Annotations** option, and then in any dashboard cell, click the point in time to add the annotation.
-  {{% note %}}
-**Tip:** To move the annotation outside of the selected graph time range, first create the annotation, and then [edit the annotation](#edit-an-annotation) to update the timestamp as needed.
-  {{% /note %}}
-5. On the **Add Annotation** page:
+2. Select an existing dashboard to add the annotation to, or [create a new dashboard](/influxdb/cloud/visualize-data/dashboards/create-dashboard/). The **Annotations** option is selected by default.
+3. In a dashboard cell, add an annotation by pressing Shift+click on a a single point in time .
```
Comment (daviddkkim) [inline content/influxdb/cloud/visualize-data/annotations.md]: typo

**Review by daviddkkim — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud/visualize-data/annotations.md (hunk 1 above)
Comment (daviddkkim) [reply content/influxdb/cloud/visualize-data/annotations.md]: there's two a's

Comment (kelseiv) [reply content/influxdb/cloud/visualize-data/annotations.md]: fixed in 4a2a19b

**Review by daviddkkim — APPROVED** [review]
LGTM

**Pull request merged**